tests/Feature/MovieTest.php

```diff
@@ -0,0 +1,235 @@
+<?php
+
+namespace Tests\Feature;
+
+use Tests\TestCase;
+use App\Models\User;
+use App\Models\Movie;
+use Illuminate\Foundation\Testing\RefreshDatabase;
+
+class MovieTest extends TestCase
+{
+    use RefreshDatabase;
+
+    /** @test */
+    public function an_admin_can_see_movies_view()
+    {
+        $admin = User::factory()->admin()->create();
+
+        $this->actingAs($admin)
+            ->get('/movies')
+            ->assertOk();
+    }
+
+    /** @test */
+    public function a_user_can_see_movies_view()
+    {
+        $user = User::factory()->create([
+            'is_admin' => false,
+        ]);
+
+        $this->actingAs($user)
+            ->get('/movies')
+            ->assertOk();
+    }
+
+    /** @test */
+    public function an_admin_can_access_create_movie_view()
+    {
+        $admin = User::factory()->admin()->create();
+
+        $this->actingAs($admin)
+            ->get('/movies/create')
+            ->assertOk();
+    }
+
+    /** @test */
+    public function a_user_cannot_access_create_movie_view()
+    {
+        $user = User::factory()->create([
+            'is_admin' => false
+        ]);
+
+        $this->actingAs($user)
+            ->get('/movies/create')
+            ->assertForbidden();
+    }
+
+    /** @test */
+    public function an_admin_can_add_a_movie()
+    {
+        $admin = User::factory()->admin()->create();
+
+        $this->actingAs($admin)
+            ->postJson('/movies', [
+                'title' => 'Sample Movie',
+                'synopsis' => 'This is a Sample Synopsis',
+                'year' => 2021,
+                'poster' => 'https://via.placeholder.com/600x750.png/00aa33?text=totam',
+                'trailer' => 'http://www.goyette.biz/',
+                'duration' => '190',
+            ])->assertOk();
+
+        $this->assertDatabaseHas('movies', [
+            'title' => 'Sample Movie',
+            'synopsis' => 'This is a Sample Synopsis',
+            'year' => 2021,
+            'poster' => 'https://via.placeholder.com/600x750.png/00aa33?text=totam',
+            'trailer' => 'http://www.goyette.biz/',
+            'duration' => '190',
+        ]);
+    }
+
+    /** @test */
+    public function a_user_cannot_add_a_movie()
+    {
+        $user = User::factory()->create([
+            'is_admin' => false
+        ]);
+
+        $this->actingAs($user)
+            ->postJson('/movies', [
+                'title' => 'Sample Movie',
+                'synopsis' => 'This is a Sample Synopsis',
+                'year' => 2021,
+                'poster' => 'https://via.placeholder.com/600x750.png/00aa33?text=totam',
+                'trailer' => 'http://www.goyette.biz/',
+                'duration' => '190',
+            ])->assertForbidden();
```

Comment:
Same as before, I believe it would make sense to assert the status. I think that it becomes a bit more clear about what error you are expecting. 

We can check with Steve what is the convention 😄